keras/src/backend/config.py

```diff
@@ -274,3 +276,23 @@ def backend():
 
     """
     return _BACKEND
+
+
+@keras_export("keras.config.enable_flash_attention")
+def enable_flash_attention(value):
+    """Enable flash attention.
+
+    Flash attention offers performance optimization for attention layers,
+    making it especially useful for large language models (LLMs) that
+    benefit from faster and more memory-efficient attention computations.
+
+    Once enabled, supported layers like `MultiHeadAttention` will
+    use flash attention for faster computations.
+    """
+    global _ENABLE_FLASH_ATTENTION
+    _ENABLE_FLASH_ATTENTION = value
```

Comment:
This is needs to be threadlocal. Instead of doing it like this, use `set_global_attribute`/`get_global_attribute` from `keras.src.backend.common.global_state`. See how other global flags are implemented.

Reply:
importing `global_state` inconfig created circular import error so I have moved the configs to attention.py